README.md

```diff
@@ -445,6 +445,34 @@ fn main() {
 }
 ```
 
+Or if you want a specific cell, you can modify `Column`/`Row`.
+
+```rust
+use tabled::{
+    object::Columns,
+    style::{BorderChar, Offset, Style},
+    Modify, Table,
+};
+
+fn main() {
+    let table = Table::new([["Hello", "World", "!"]])
+        .with(Style::markdown())
+        .with(
+            Modify::new(Columns::single(1))
+                .with(BorderChar::horizontal(':', Offset::Begin(0)))
+                .with(BorderChar::horizontal(':', Offset::End(0))),
+        )
+        .to_string();
+
+    assert_eq!(
+        table,
+        "| 0     | 1     | 2 |\n\
+         |-------|:-----:|---|\n\
+         | Hello | World | ! |"
+    );
+}
+```
```

Comment:
It seems to be a copy&paste from the above?
But it shall not have `Modify`, right?

Reply:
My bad, I have fixed it now. The intention is to show the difference between Table level vs specific Row/Column.